app/helpers/reports/regions_helper.rb

```diff
@@ -21,6 +21,13 @@ def sum_registration_counts(repository, *keys)
       .flatten.compact.sum
   end
 
+  def sum_bp_measures(repository, *keys)
```

Comment:
I would probably just name slug and user_id as params and skip the splat/decompose. Looks like you followed the pattern of the above method, but I don't know why the above method is doing it either.

Reply:
I believe the idea was there are other keys we may split on.

I think I'll refactor this into the repository, so not too worried about these helpers being a little bit weird for right now.